js/main.js

```diff
@@ -21,4 +26,79 @@ $form.addEventListener('submit', function (e) {
   data.entries.unshift(object);
   $image.src = 'images/placeholder-image-square.jpg';
   $form.reset();
+
+  $dataViewEntries.appendChild(renderEntry(object));
```

Comment:
You should be using `prepend` here rather than `appendChild` so it gets added to top of the page